app/src/components/healthSvc.js

```diff
@@ -0,0 +1,37 @@
+const axios = require('axios');
+const config = require('config');
+const log = require('npmlog');
+
+const utils = require('./utils');
+
+const healthSvc = {
+  /**
+   *  @function check
+   *  Call the health checks.
+   *  1. Call logstash node stats events API to see if we can connect to logstash
+   *  Return health check object array
+   */
+  check: async () => {
+    const logstash = {
+      name: 'logstash',
+      healthy: false,
+      info: ''
+    };
+    try {
+      const response = await axios.get(config.get('elkStack.logstashUrl'));
```

Comment:
bit late here and i'm probably missing something here but are you calling the logstashUrl ? isnt the node stats events API at /_node/stats/events?pretty'

Reply:
yeah, good catch.  will have to change that comment next PR.
i was going to hit stats, but the configured logstash url is to hit the http port, and we dont have the exposed _node/stats.  So hitting the /http is good enough for now to see that we can connect.

Reply:
yeah i dont see the difference